math/divisor.cpp

```diff
@@ -0,0 +1,97 @@
+/**
+ * @file
+ * @brief Algorithms to find divisor of all natural numbers.
+ * @author [Tee Chia Suen](https://github.com/ChiaSuen)
+ */
+
+#include <cassert>  /// for assert
+#include <iostream>
+#include <vector>  //for declaring undefined size array
+
+/**
+ * @namespace <check from other files in this repo>
+ */
+namespace math {
+
+/**
+ * @brief Function to calculate the divisors and append it to a vector array
+ * @tparam int
+ * @param Number to calculate the divisors
```

Comment:
```suggestion
 * @param n number to calculate the divisors
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix